packages/repository/src/__tests__/acceptance/has-many.relation.acceptance.ts

```diff
@@ -152,6 +152,35 @@ describe('HasMany relation', () => {
     ).to.be.rejectedWith(/`orders` is not defined/);
   });
 
+  // The following tests test hasMany Entity to itself
```

Comment:
We use `context` to group similar tests together. That way it's clearer where the group starts and where it ends.

```ts
context('when targeting the source model', () => {
  it('gets the parent entity through the child entity', /*...*/);
  
  it('creates a child entity through the parent entity', /*...*/);
});
```